UIDokan/Sources/UIDokan/CustomTemplates/AlertController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  File.swift
+//
+//
+//  Created by ziad on 04/08/2022.
+//
+
+import UIKit
+
+/*
+ * This extension implements a Builder pattern to use with the UIAlertViewController.
+ * Sample:
+ * UIAlertController.Builder()
+ *    .withTitle("this_is_a_key_from_localizable.strings")
+ *    .withMessage("This is a normal message")
+ *    .addOkAction()
+ *    .show(in: UIViewController) OR .build() to get back the UIAlertController directly
+ */
+
+extension UIAlertController {
```

Comment:
```suggestion
/// This extension implements a Builder pattern to use with the UIAlertViewController.
/// Example:
/// ```
/// UIAlertController.Builder()
///    .withTitle("this_is_a_key_from_localizable.strings")
///    .withMessage("This is a normal message")
///    .addOkAction()
///    .show(in: UIViewController) OR .build() to get back the UIAlertController directly
/// ```
extension UIAlertController {
```